### DISCUSSION TITLE
Deleting archive - nothing is being deleted

### DISCUSSION THREAD

**Discussion by DrMartinus** [borgbase/vorta#2064] [General]
I am using vorta because I prefer the GUI approach. I had with a former backup a problem that I deleted all the files from the target HDD and then couldn't start a new archive because Vorta insisted that there was an old archive. I found then that I can delete the archive from within Vorta. This I did now in order to be sure everything goes the proper way. I saw it accessing the HDD continuously for about three hours (the archive is about 2 TB), after which it stopped saying that the archive was deleted. But on the HDD there are still all files. There are still about 2TB occupied. Is the archive not what is on the HDD?
Can I now safely delete all the files on the HDD and start a new backup?
I like borg/vorta a lot, it does its job quickly and needs less space, that's always good. But somehow I don't seem to have grasped its working philosophy.
Sorry if this has been asked and answered before, I didn't find anything in this regard.

**Comment by m3nu** [comment]
You may need to compact the repo after deleting a large archive.

If you have enough space, you could also start over with a new repo and keep the old one around just in case.

**Reply by DrMartinus** [reply]
Thank you for your reply. I guess I don't grasp the concept yet. I deleted all files on the HDD manually in order to put the new repo there. There wouldn't have been enough space for two such repos. I notice that there seems to be a difference between archive and repo? From previous backups (before borg) it was my understanding, that the archive consisted of all the backed up files.
Anyway, quickly another question: Is there a way to do incremental backups? I didn't see an option for that yet.

**Reply by m3nu** [reply]
Every backup will be incremental. If a file already exists in a repo, the data won't be added again. Here the concepts you asked about:

- Repo: A folder on disk that has multiple archives. This is where all data is stored. Can't be used directly without Borg.
- Archive: One or more source folders at a specific point in time. Sometimes called "snapshot". 

It may help to think about a repo as a collection of file contents. And an archive structures them into where a file was at some point in time. E.g. "blah.jpeg as it was 2 weeks ago". So archives are like a folder structure with an added time dimension.
